src/Synapse/Synapse/help/Get-AzSynapseDroppedSqlPool.md

```diff
@@ -0,0 +1,115 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Synapse.dll-Help.xml
+Module Name: Az.Synapse
+online version: https://docs.microsoft.com/en-us/powershell/module/az.synapse/get-azsynapsedroppedsqlpool
+schema: 2.0.0
+---
+
+# Get-AzSynapseDroppedSqlPool
+
+## SYNOPSIS
+Gets a dropped Sql pool backup of a Synapse Sql Pool.
+
+## SYNTAX
+
+### GetByNameParameterSet (Default)
+```
+Get-AzSynapseDroppedSqlPool [-ResourceGroupName <String>] -WorkspaceName <String> -Name <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
```

Comment:
Please also provide getbyresourceId parameter set for Get-AzSynapseDroppedSqlPool

Reply:
I think we dont have it when I refer to sql commands. Please feel free to let me know if I'm wrong